app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -215,8 +215,8 @@ fun CommentOrPostNodeHeader(
                 bottom = MEDIUM_PADDING
             )
             .combinedClickable(
-                onLongClick = onLongClick,
-                onClick = {}
+                onLongClick = {},
```

Comment:
Could you create a github issue for long-click voting? I think that'd be really nice to have.

Reply:
https://github.com/dessalines/jerboa/issues/418